src/components/Recipe.js

```diff
@@ -75,22 +93,7 @@ export default function Recipe() {
     fetchRecipe();
   }, []);
 
-  // let arr = [];
-  // for (let i = 1; i <= 20; i++) {
-  //   arr.push(`strIngredient${i}` + " ");
-  // }
-
-  const items = [];
-
-  for (let i = 1; i <= 20; i += 1) {
-    const strIngredient = [`strIngredient${i}`];
-    const strMeasure = [`strMeasure${i}`];
-
-    items.push({
-      ingredient: strIngredient,
-      measure: strMeasure,
-    });
-  }
+  const items = createIngredientList(recipe);
```

Comment:
I moved the logic into a separate function which makes it easier to test and move between files, if needed